internal/manifest/manifest.go

```diff
@@ -49,6 +49,9 @@ func LoadManifest(filePath string) (*Manifest, error) {
 	var m Manifest
 
 	if _, err := toml.DecodeFile(filePath, &m); err != nil {
+		// TODO: consider which error to return when loading of deprecated
```

Comment:
TODO Before or after release of feature?

Reply:
I think we can leave this for now. This is not new behaviour, but I just noticed it while testing this feature, and wanted to document it.